python/lsst/ts/scriptqueue/base_script.py

```diff
@@ -43,6 +46,25 @@ class ScriptState(enum.IntEnum):
     """Script exiting after an error."""
 
 
+class ScriptProcessState(enum.IntEnum):
+    """processState constants.
+    """
+    UNKNOWN = 0
+    """Script process state is unknown."""
```

Comment:
Again what is the need for UNKNOWN? I don't see any use in the code (except one commented-out block that doesn't look like it really needs it). I suggest sticking values that have a SALPY enum equivalent unless there is a strong need to do otherwise.